pkg/kube/secrets.go

```diff
@@ -187,5 +187,16 @@ func (r *secretsReader) CredentialsByWorkload(ctx context.Context, workload clie
 	if err != nil {
 		return nil, err
 	}
+
+	for ns, secretName := range secretsInfo {
+		var envSecret corev1.Secret
+		err = r.client.Get(ctx, client.ObjectKey{Name: secretName, Namespace: ns}, &envSecret)
+
+		if err != nil && !k8sapierror.IsNotFound(err) {
+			return nil, fmt.Errorf("getting secret by name: %s/%s: %w", ns, secretName, err)
+		}
+
+		imagePullSecrets = append(imagePullSecrets, envSecret)
+	}
```

Comment:
nit: maybe extract it to method

Reply:
Chen, hi! Added separate method - `GetSecretsFromEnv`